src/Sulu/Bundle/AdminBundle/Gruntfile.js

```diff
@@ -121,25 +137,47 @@ module.exports = function(grunt) {
                 files: [
                     {src: ['Resources/public/dist/app.min.js'], dest: 'Resources/public/dist/app.min.js'}
                 ]
+            },
+            buildLogin: {
+                options: {
+                    variables: {
+                        '/js/login.js': '/dist/login.js'
+                    },
+                    prefix: ''
+                },
+                files: [
+                    {src: ['Resources/public/dist/login.min.js'], dest: 'Resources/public/dist/login.min.js'}
+                ]
             }
         },
         requirejs: {
-            compile: {
-                options: {
-                    baseUrl: 'Resources/public/js/',
-                    mainConfigFile: 'Resources/public/js/main.js',
-                    preserveLicenseComments: false
+            app: {
+                compile: {
+                    options: {
+                        baseUrl: 'Resources/public/js/',
+                        mainConfigFile: 'Resources/public/js/main.js',
+                        preserveLicenseComments: false
+                    }
+                }
+            },
+            login: {
```

Comment:
i suppose that the reason why the login.min.js file is always empty is somewhere in this block